toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -156,7 +156,7 @@ func createOverlayPartition(partitionSetting configuration.PartitionSetting, mou
 	//Mount the base image
 	//Create a temp upper dir
 	//Add to the mount args
-	devicePath, err := diskutils.SetupLoopbackDevice(partitionSetting.OverlayBaseImage)
+	devicePath, err := diskutils.SetupLoopbackDevice(partitionSetting.OverlayBaseImage, nil)
```

Comment:
You probably want to make sure that the overlay image is using the same block size as the disk it is overlaying.

Reply:
I don't think that is a requirement.

Disk block size is independent of file system block size. But it shouldn't matter as overlay mounts don't even have to use the same file system let alone the file system block size.